CONTRIBUTING.md

```diff
@@ -20,7 +20,8 @@ Press F5. VSCode should build and start the extension.
 ## Installing the extension globally
 
 ```sh
-yarn package
+yarn esbuild
```

Comment:
```suggestion
```
There's no need to run esbuild beforehand, `vsce:package` does it automatically :)